changes/5592.fixed

```diff
@@ -0,0 +1 @@
+Fixed plugins not loading when using Gunicorn.
```

Comment:
Anything we can/should add from a unit test perspective here?

Reply:
What are you thinking? Calling `nautobot.core.uwsgi` and checking for setup being called is the only viable test I can think of

Reply:
Was hoping that we could do something similar to the original issue, maybe calling gunicorn via `subprocess` or something? If not, no worries.